File: add/codesnippet/VisualBasic/p-system.windows.forms.l_27_1.vb
    Private list As New ImageList()
    Private hotTrackinglistView As New ListView()


    Private Sub InitializeHotTrackingListView()
        list.Images.Add(New Bitmap(GetType(Button), "Button.bmp"))
        hotTrackinglistView.SmallImageList = list
        hotTrackinglistView.Location = New Point(20, 20)
        hotTrackinglistView.View = View.SmallIcon
        Dim listItem1 As New ListViewItem("Short", 0)
        Dim listItem2 As New ListViewItem("Tiny", 0)
        hotTrackinglistView.Items.Add(listItem1)
        hotTrackinglistView.Items.Add(listItem2)
        hotTrackinglistView.HotTracking = True
        Me.Controls.Add(hotTrackinglistView)

    End Sub

File: add/codesnippet/CSharp/p-system.windows.forms.l_27_1.cs
        private ImageList list = new ImageList();
        private ListView hotTrackinglistView = new ListView();

        private void InitializeHotTrackingListView(){
            list.Images.Add(new Bitmap(typeof(Button), "Button.bmp"));
            hotTrackinglistView.SmallImageList = list;
            hotTrackinglistView.Location = new Point(20, 20);
            hotTrackinglistView.View = View.SmallIcon;
            ListViewItem listItem1 = new ListViewItem("Short", 0 );
            ListViewItem listItem2 = new ListViewItem("Tiny", 0);
            hotTrackinglistView.Items.Add(listItem1);
            hotTrackinglistView.Items.Add(listItem2);
            hotTrackinglistView.HotTracking = true;
            this.Controls.Add(hotTrackinglistView);
        }